gl_extensions/include/core/gl_extensions.hpp

```diff
@@ -1,13 +1,6 @@
+///TODO: consider accumulating fnction pointers into this class, and calling corresponding extension functions through it
```

Comment:
I presume you should do it now.

Reply:
This might get tricky enough to be a separate task

Reply:
Yeah overall this task is complex enough to be a separate one.

Reply:
'Kay, do not forget to create this task